src/utils/testHelpers.ts

```diff
@@ -82,9 +82,8 @@ export function mockAPICalls(testName: string, url = MainnetUrl) {
  * or the one with attached INFURA SECRET key from
  * UNSTOPPABLE_RESOLUTION_INFURA_PROJECTID env variable if any
  */
-export function secretInfuraLink(): string {
+export function secretInfuraLink(options: {wss: boolean} = {wss: false}): string {
```

Comment:
I think we'd better use enum here as well, maybe in future we'll see some other protocols emerging